compiler/src/main/java/com/squareup/anvil/compiler/codegen/BindingUtils.kt

```diff
@@ -0,0 +1,23 @@
+package com.squareup.anvil.compiler.codegen
```

Comment:
Good question. I agree but was also conflicted because some extensions are specific to certain annotations, like `ClassReference.checkNotMoreThanOneQualifier` is specific to ContributesBinding/ContributesMultibinding. I suppose for that one we could remove the comment about those two annotations or rename it to make it more obvious? or maybe it's just fine if some extensions aren't generic

Reply:
I made a small adjustment to the comment in that method which I think makes it feel better, and went ahead with renaming the file to `ClassReferenceExtensions.kt`